odl/contrib/tomo/elekta.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Tomography helpers for Elekta systems."""
+
+import numpy as np
+import odl
```

Comment:
As long as the `odl/contrib/__init__.py` does nothing there is no problem since `import odl` doesn't care about anything in `odl/contrib`. So I currently don't see a way this can go wrong.